core/src/main/java/io/questdb/cairo/TableReaderMetadata.java

```diff
@@ -152,12 +152,15 @@ public void dumpTo(MemoryMA mem) {
 
     @Override
     public void close() {
-        // TableReaderMetadata is re-usable after close, don't assign nulls
```

Comment:
nit: I'd keep this comment since it's still relevant.

Reply:
restored